lib/cronex/description/base.rb

```diff
@@ -13,7 +13,7 @@ def special_chars
     end
 
     def segment_description(expression, all_values_description)
-      if expression.empty? || expression == '0'
+      if expression.empty? || (expression == '0' && !self.class == Cronex::HoursDescription)
```

Comment:
I'd go with `&& self.class != Cronex::HoursDescription`

Reply:
Sure, I will update.

Reply:
Updated the syntax.